### PR TITLE
Bugfix/change 404 component style

### PR DESCRIPTION
**Pull request by martyniukroman**

### CODE REVIEW

**Review by SergiyKozachok — APPROVED** [review]

File: src/app/Components/Layout/footer/footer.component.html [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
-        <div class="footer-contacts" fxLayout="row" fxLayoutGap="5px" fxLayout="column">
-            <h2>Contacts</h2>
-            <div>
-                <p>Rivne Office</p>
-                <p>Rivne, Slovatskoho Str. 4/6, floor 2</p>
-                <p>+38 (096) 549 7678</p>
-            </div>
-        </div>
+<footer fxFlexAlign="end" id="FatherBox" fxLayout="row" fxLayoutAlign="space-between center">
+
```
Comment (TREN-D) [inline src/app/Components/Layout/footer/footer.component.html]: Remove space

File: src/app/Components/Layout/footer/footer.component.html [hunk 2]
```diff
@@ -1,35 +1,31 @@
...
+            <i class="material-icons">announcement</i> Signal
+        </a>
+        <a fxHide fxShow.md fxShow.lg class="link-element" mat-button fxLayout="row" fxLayoutAlign="center center">
+            <i class="material-icons">local_phone</i>
+            <span id="phone-number">
+                +38 (096) 549 7678
+            </span>
+        </a>
+    </div>
+
```
Comment (TREN-D) [inline src/app/Components/Layout/footer/footer.component.html]: Here too

**Review by TREN-D — APPROVED** [review]
Next time name your branch/pull request correct

File: src/app/Components/Layout/footer/footer.component.html (hunk 1 above)
Comment (martyniukroman) [reply src/app/Components/Layout/footer/footer.component.html]: done

File: src/app/Components/Layout/footer/footer.component.html (hunk 2 above)
Comment (martyniukroman) [reply src/app/Components/Layout/footer/footer.component.html]: done

**Review by edward0802 — APPROVED** [review]

**Review by eugent482 — APPROVED** [review]

**Pull request merged**